### PR TITLE
Model genre

### PR DESCRIPTION
**Pull request by 15Max**
-Modified the utils/models.py file by adding the first version of the genre_model (accuracy 60%), this part might be changed if I find a better performing model, but I won't modify the model name, so the integration process with the webapp can stay the same 
- Added a genre.pth file containing the weights of the first genre model
- Added the notebook to train the model, it's structured in a similar way to the one for danceability, just added some plots and also computed the accuracy on top of the loss.
- I used a different loss function (cross entropy) since this is a classification problem, not a regression problem
- I also used Neptune for tuning, but I created a different project to avoid any confusion with the danceability model

### CODE REVIEW

**Comment by 15Max** [comment]
@MadelineKelly196 if it is useful to you I created the merge request for the 'minimum viable model'
